task/01-strings-tasks.js

```diff
@@ -200,8 +212,17 @@ function extractEmails(str) {
  *             '└──────────┘\n'
  *
  */
+
 function getRectangleString(width, height) {
-    throw new Error('Not implemented');
+    let normalizeWidth, normalizeHeight;
+    width <=2 ? normalizeWidth = 0 : normalizeWidth = width - 2;
```

Comment:
never use asignment inside ternary operator. instead of this `condition ? res = val1 : res = val2` do this `res = conditions ? val1 : val2`